Modules/posixmodule.c

```diff
@@ -174,6 +174,7 @@ corresponding Unix manual entries for more information on calls.");
 #define HAVE_FSYNC      1
 #define fsync _commit
 #else
+#ifndef __VXWORKS__
```

Comment:
Are all of this function absent? It looks suspicious that all of them are grouped alphabetically.

Reply:
A couple are present in some configurations of VxWorks, but following comment is dated.
"Unix functions that the configure script doesn't check for"
The automake configure script finds all these without issue. So a better change would be to simply remove all these and let automake do it's job. I'm just not prepared to regression test the 1/2 dozen other OS referenced in this section.